sdks/android/Mentat/library/src/main/java/org/mozilla/mentat/RustObject.java

```diff
@@ -27,9 +27,7 @@
 abstract class RustObject implements AutoCloseable {
     // This should probably be private to let us better prevent usage mistakes (which lead to
     // memory-unsafety).
-    protected Pointer rawPointer;
-    // If this is set to true, we don't destroy this object. Right now this is used for TypedValueAtIndex
-    boolean notOwned = false;
+    private Pointer rawPointer;
```

Comment:
Ah, you addressed my concern.  Please fold this follow-up back into your stack before landing as well.